platform/android/export/export.cpp

```diff
@@ -2536,6 +2576,22 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 				EditorNode::get_singleton()->show_warning(TTR("Building of Android project failed, check output for the error.\nAlternatively visit docs.godotengine.org for Android build documentation."));
 				return ERR_CANT_CREATE;
 			}
+
+			if (export_format == 1) {
+				List<String> copy_args;
+				String variant = p_debug ? "debug/build-debug" : "release/build-release";
+				String output_path = vformat("android/build/build/outputs/bundle/%s.aab", variant);
+
+				copy_args.push_back(ProjectSettings::get_singleton()->get_resource_path().plus_file(output_path));
+				copy_args.push_back(p_path);
+				int copy_result = EditorNode::get_singleton()->execute_and_show_output(TTR("Moving output"), copy_command, copy_args);
+				if (copy_result != 0) {
+					EditorNode::get_singleton()->show_warning(TTR("Unable to copy and rename export file, check gradle project directory for outputs."));
+					return ERR_CANT_CREATE;
+				}
+				return OK;
+			}
```

Comment:
What's the copy logic for?

Reply:
The generated AAB is stored at `res://android/build/build/outputs/bundle/debug/build-debug.aab` because that's the default location Gradle exports to, so we have to copy the AAB to the location the user requests.